dao/src/main/java/com/redhat/ipaas/dao/manager/DataManager.java

```diff
@@ -140,13 +142,14 @@ public String generatePK(Cache<String, WithId> entityMap) {
 
     @SuppressWarnings("unchecked")
     public <T extends WithId> ListResult<T> fetchAll(Class<T> model, Function<ListResult<T>, ListResult<T>>... operators) {
-        Kind kind = Kind.from(model);
-        Cache<String, WithId> cache = caches.getCache(kind.getModelName());
 
-        ListResult<T> result = ListResult.of((Collection<T>) cache.values());
-         
-        if( result == null ) {
+        ListResult<T> result;
+        if( getDataAccessObject(model)!=null ) {
```

Comment:
Who knows what we want at scale? It all needs revisiting post-summit. I'd even consider a distributed cache that we use for persistence, potentially even obviating the need for PostgreSQL...

Regardless, right now we're not using the cache properly anyway so let's merge this and revisit it later.